XVim/Test/XVimTestCase.m

```diff
@@ -68,6 +69,7 @@ - (void)setUp{
     [[[XVimLastActiveSourceView() xvim_window] sourceView] xvim_changeSelectionMode:XVIM_VISUAL_NONE];
     [XVimLastActiveSourceView() setString:self.initialText];
     [XVimLastActiveSourceView() setSelectedRange:self.initialRange];
+    [XVim instance].options.expandtab = YES;
```

Comment:
I will see if I can remove it.

I was not sure if options get reset to default between tests. If they are reset to default, then it is not necessary.